homeassistant/components/openalpr_local/image_processing.py

```diff
@@ -130,14 +130,14 @@ def extra_state_attributes(self):
         """Return device specific state attributes."""
         return {ATTR_PLATES: self.plates, ATTR_VEHICLES: self.vehicles}
 
-    def process_plates(self, plates, vehicles):
+    def process_plates(self, plates: dict[str, float], vehicles: int) -> None:
         """Send event with new plates and store data."""
         run_callback_threadsafe(
             self.hass.loop, self.async_process_plates, plates, vehicles
         ).result()
 
     @callback
-    def async_process_plates(self, plates, vehicles):
+    def async_process_plates(self, plates: dict[str, float], vehicles: int) -> None:
```

Comment:
A TypedDict could be useful here, too.

Reply:
Good point! Could you change the docstring to make that a bit more obvious?
```diff
-          { 'plate': confidence }
+          { '<plate>': confidence }
```

Reply:
done